packages/graphback/src/generators/resolvers/services/ServiceRuntimeResolverGen.ts

```diff
@@ -0,0 +1,72 @@
+import { getFieldName, getTableName, ResolverType } from '../../..'
+import { Type } from '../../../input/ContextTypes'
+import { GraphbackCRUDService } from '../../../layers/service/GraphbackCRUDService'
+
+/**
+ * Generate runtime resolver layer using Apollo GraphQL format 
+ * and injected service layer. Service layer offers various capabilities like monitoring, cache etc. 
+ * so resolver logic can be kept simple and interchangable.
+ */
+export class ServicesRuntimeResolverGenerator {
```

Comment:
@b1zzu You would love this. 
This will create a runtime resolver layer for testing. 
I created that first for building a suite of integration tests.
Going to push more examples

Reply:
I think the ResolverGenerator should receive a more advanced `inputContext` that contains also the fields names and the table names because both TS and Runtime resolvers will need them

Reply:
Same for all of the `if config.create`, `if config.update`, `...` the config should be applied outside the Generator and the generator should only loop over the resolvers.

Reply:
I've created a dummy example: [LayeredRuntimeResolverGen.ts](https://github.com/aerogear/graphback/blob/alternative-runtime-resolver/packages/graphback/src/generators/resolvers/layered/LayeredRuntimeResolverGen.ts)